mongofiles/mongofiles_test.go

```diff
@@ -8,25 +8,26 @@ package mongofiles
 
 import (
 	"bytes"
+	"context"
 	"fmt"
+	"github.com/mongodb/mongo-go-driver/bson/primitive"
+	"github.com/mongodb/mongo-go-driver/mongo/gridfs"
 	"io/ioutil"
 	"os"
 	"strings"
 	"testing"
 
-	"github.com/mongodb/mongo-tools/common/db"
-	"github.com/mongodb/mongo-tools/common/json"
-	"github.com/mongodb/mongo-tools/common/log"
-	"github.com/mongodb/mongo-tools/common/options"
-	"github.com/mongodb/mongo-tools/common/testtype"
-	"github.com/mongodb/mongo-tools/common/testutil"
-	"github.com/mongodb/mongo-tools/common/util"
+	"github.com/mongodb/mongo-tools-common/db"
+	"github.com/mongodb/mongo-tools-common/log"
+	"github.com/mongodb/mongo-tools-common/options"
+	"github.com/mongodb/mongo-tools-common/testtype"
+	"github.com/mongodb/mongo-tools-common/testutil"
+	"github.com/mongodb/mongo-tools-common/util"
 	. "github.com/smartystreets/goconvey/convey"
-	"gopkg.in/mgo.v2"
 )
 
 var (
-	testDB     = "mongofiles_test_db"
+	testDBName = "mongofiles_test_db"
```

Comment:
This is another minor renaming that possibly should be part of a different pull request.  Sometimes renaming as you go is good for clarity -- as this is in abstract -- but seeing this or "data" vs "Data" below adds to the cognitive burden of code review.  The goal should be simplifying CR, so anything unrelated can/should be part of a separate commit.

Reply:
Reverted both this and the "data" one (I believe the latter was a mistake).

I will keep that in mind going forward and reign in some of the more trivial refactors / changes in later ports / CRs.